.dev_scripts/gather_models.py

```diff
@@ -212,11 +183,15 @@ def convert_model_info_to_pwc(model_infos):
 def parse_args():
```

Comment:
Have you tested epoch-based and iteration-based methods?

Reply:
Yep